src/playground/service.clj

```diff
@@ -60,19 +63,39 @@
 ;;auth interceptors
 
 (def session-auth-backend
-  (session-backend
+  (basic
```

Comment:
Basic is this https://en.wikipedia.org/wiki/Basic_access_authentication

It's not what we want, since it's a quite primitive and insecure form of auth.